ahcer/src/app/veiw-episode/view-episode.component.ts

```diff
@@ -13,7 +13,7 @@ export class ViewEpisodeComponent implements OnInit {
   episode: Episode;
   patientId: string;
   rescueMeds: Medication[]=[];
-  rescueMedsDoses: Object={};
+  rescueMedsDosesAndTimes: Object={};
```

Comment:
Line 8 'view' is misspelled.

Reply:
I keep forgetting to fix that, but it should be fixed now!